src/buttons/ButtonGroup.js

```diff
@@ -95,7 +98,10 @@ const ButtonGroup = props => {
                 borderBottomLeftRadius: containerBorderRadius,
               },
               isSelected && {
-                backgroundColor: colors.primary,
+                backgroundColor:
+                  (buttonStyleObj && buttonStyleObj.backgroundColor) ||
+                  (containerStyleObj && containerStyleObj.backgroundColor) ||
+                  colors.primary,
               },
```

Comment:
I think you just need to move the
```
isSelected && {
  backgroundColor: colors.primary,
},
```

from the `Component` to the `View` just below ([line 102](https://github.com/react-native-training/react-native-elements/pull/1040/files#diff-6bc2a6e42a02b3c01d6f4ccaf34c94cfL102))

Reply:
Thanks for the warning. fixing it right away.